<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>errifierui</class>
 <widget class="QMainWindow" name="errifierui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Errifier</string>
  </property>
  <property name="windowIcon">
   <iconset resource="errifier.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="txt_log">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Log Location</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_friendlyname">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Friendly Name</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="ping">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ping</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Watch</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serverlist">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>List of Servers</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>701</width>
      <height>401</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAdd_Server">
   <property name="text">
    <string>Add Server</string>
   </property>
  </action>
  <action name="actionDelete_Server">
   <property name="text">
    <string>Delete Server</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="errifier.qrc"/>
 </resources>
 <connections/>
</ui>
